packages/client/styles/paletteV2.ts

```diff
@@ -1,6 +1,13 @@
 export const enum PALETTE {
   BACKGROUND_MAIN = '#F1F0FA',
+  BACKGROUND_MAIN_70 = '#F5F5FC',
+  BACKGROUND_PRIMARY = '#493272',
+  BACKGROUND_PRIMARY_70 = '#80709C',
+  BACKGROUND_PRIMARY_10A = '#4932721A',
+  BACKGROUND_PRIMARY_20A = '#49327233',
```

Comment:
definitely not a show stopper, but to improve DX, we should name these as to how they should be used. For example, if you called this `BACKGROUND_INPUT_PLACEHOLDER` it'd be a lot harder to misuse.

As a user, if i saw a 20% alpha next to a 30%, i'd probably think it was a bug because it's only _slightly_ off. Ideally, we could merge a BUNCH of these similar colors & then rename them to something a little more meaningful.

Reply:
I agree with finding a better semantic. As for the subtle variations, they’re necessary for the finer details of various interactions.

I’m struggling with things like `BACKGROUND_BLUE` and `BORDER_BLUE` and `TEXT_BLUE` if they’re all plain ol’ palette blue. I think if there’s a good semantic based on purpose, scoped to a property, then that works. The structure would be `[PROPERTY]_[PURPOSE]`. I’d opt for those variants first. However, if the purpose is vague or general in some cases and we need a variant, I’d just use `BLUE` or `BLUE_LIGHTER`. As for the number values like 10, 20, 30, I know what the color is in Figma and I could scope it to a better semantic later, scrubbing the codebase. Meh! I’ll make a pass!

Reply:
i'm OK with 2 enums sharing the same underlying value. I just tell myself that it's just a coincidence that BACKGROUND_BLUE is the same blue as TEXT_BLUE. After all, if it were PLACEHOLDER_BLUE it'll probably be a different blue.

My preference is to avoid the trap of LIGHT/LIGHTER/LIGHTEST since those don't tell me when to use em. if 1 button is BLUE and it should light up, I'd much rather call it BUTTON_BLUE_HOVERED.

expanding on your idea, i'd suggest [PROPERTY]_[COLOR]?_[PURPOSE]. 
e.g. we create a new green button & assign it BUTTON_GREEN. Then, when we add hover capability to it, we create a new BUTTON_GREEN_HOVERED. if it were a scrim we could have SCRIM_PRIMARY and then if it were a modal that you can't close we could call it SCRIM_BLACK_FORCED. 

PROPERTY would be something like BACKGROUND, SCRIM, BORDER, CONTROL, TEXT, etc.
COLOR could be PRIMARY, ERROR, YELLOW, BLUE, RED, TEAL, GREEN, etc
PURPOSE could be an optional suffix when the first 2 aren't discriminatory enough